src/components/projects/new/edit-activity.svelte

```diff
@@ -327,6 +327,11 @@
             toggle(e);
             fieldsAltered = true;
           }}
+          on:change-date={({detail}) => {
```

Comment:
Put spaces around the key in the object, for consistency with the rest of the codebase